SU-SWS/stanford_basic
# D8CORE-644: Integrated mastead.

# READY FOR REVIEW

# Summary
- Updates the global header area to use Decanter's masthead behavior of lockup/search/nav.
- Removed the line-length file as it has been slated for removal and was interfering with this change.

# Review By
- October 25

# Urgency
- High, needed for Pilot.

# Steps to Test

1. Check out this branch
2. Run `nvm use`
3. Run `npm install`
4. Run `npm run publish`
5. Build a new Drupal install `blt drupal:install`
6. Review menu aesthetics at each breakpoint. 

# Affected Projects or Products
- Pilot

# Associated Issues and/or People
- D8CORE-644
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
